### DISCUSSION TITLE
Naming convention

### DISCUSSION THREAD

**Discussion by Zeinab-HasasZiabari** [Zeinab-HasasZiabari/Tafe24S1T102_Vir#4] [Q&A] [answered]
What naming convention standard are we going to use in this project? Is there any documentation?

**Comment by zlaic** [comment, marked as answer]
For naming convention, we should adhere to the ITWorks standard, utilizing the guidelines outlined in the documents "ITWorks_Coding_Standards_xaml_C#_NET.docx" for XAML and C#.NET coding standards, and "ITWorks Programming Guidelines for all languages.docx" for general programming conventions across all languages.
[ITWorks_Coding_Standards_xaml_C#_NET.docx](https://github.com/Zeinab-HasasZiabari/Tafe24S1T102_Vir/files/14644279/ITWorks_Coding_Standards_xaml_C._NET.docx)
[ITWorks Programming Guidelines for all languages.docx](https://github.com/Zeinab-HasasZiabari/Tafe24S1T102_Vir/files/14644285/ITWorks.Programming.Guidelines.for.all.languages.docx)
